### ISSUE TITLE
Port Gish to Android OS and other improvents

### ISSUE THREAD

**Issue by EXL** [blinry/gish#12]
Hello! I recently ported Gish to the Android OS and added some improvements and fixed some bugs.

https://github.com/EXL/Gish (APK in Releases)

Now Gish works through the SDL2 library. 

Building for GNU/Linux:

```sh
mkdir build; cd build
cmake -DCMAKE_BUILD_TYPE=Release -DGLES=Off ../gish/src/main/cpp/
make
```
Building for Android OS:

```sh
ANDROID_HOME="/opt/android-sdk/" ./gradlew assembleDebug
```

Video: https://youtu.be/GyMU2oV2LI4
Some info: http://exlmoto.ru/gish-droid (in Russian)

I would be glad if you integrate some improvements (for example, port on SDL2) into this repository.

Best regards!

**Comment by RblSb** [comment]
It's just INCREDIBLE !!!
(Ran to copy the resources of the game from the PC)

Upd: Nice, But for example at the custom level airship1 fps around 15. Have ideas for optimizing rendering?

**Comment by EXL** [comment]
> Upd: Nice, But for example at the custom level airship1 fps around 15. Have ideas for optimizing rendering?

@RblSb Yes, some custom levels  have many effects and therefore slow. You can try setting the "Zoom scale" parameter to 7.0 or 6.0. This should help, but it will reduce the viewport.
